src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyoutSubItem.cs

```diff
@@ -1,47 +1,888 @@
-#pragma warning disable 108 // new keyword hiding
-#pragma warning disable 114 // new keyword hiding
+using System;
+using System.Collections.Generic;
 using System.Collections.ObjectModel;
+using System.Diagnostics;
+using Uno.Disposables;
+using Windows.Foundation;
+using Windows.System;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Input;
 using Windows.UI.Xaml.Markup;
+using Windows.UI.Xaml.Media.Animation;
 
 namespace Windows.UI.Xaml.Controls
 {
 	[ContentProperty(Name = "Items")]
-	public  partial class MenuFlyoutSubItem : MenuFlyoutItemBase
+	public partial class MenuFlyoutSubItem : MenuFlyoutItemBase, ISubMenuOwner
 	{
-		public MenuFlyoutSubItem()
-		{
-			Items = new ObservableCollection<MenuFlyoutItemBase>();
-		}
+		// Popup for the MenuFlyoutSubItem
+		Popup m_tpPopup;
+
+		// Presenter for the MenuFlyoutSubItem
+		Control m_tpPresenter;
+
+		// In Threshold, MenuFlyout uses the MenuPopupThemeTransition.
+		Transition m_tpMenuPopupThemeTransition = null;
+
+		// Event pointer for the Loaded event
+		// IDisposable m_epLoadedHandler;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5205492)